.github/workflows/main.yml

```diff
@@ -126,7 +128,8 @@ jobs:
         # the next command might be needed for foxy distro, since this package is not installed
         # by default in ubuntu 20.04. For other distro, the apt install command will be ignored.
         sudo apt install -y ros-${{matrix.ros_distro}}-sensor-msgs-py
-        python3 src/realsense-ros/realsense2_camera/scripts/rs2_test.py non_existent_file
+        source ../.venv/bin/activate
+        ../.venv/bin/python3 src/realsense-ros/realsense2_camera/scripts/rs2_test.py non_existent_file
```

Comment:
I think we can use `python3` instead of `../.venv/bin/python3`, because the venv is already activated in previous line and no need to point the venv location here.